Calculator.cs

```diff
@@ -0,0 +1,25 @@
+using CalcClassLibrary;
+
+namespace Caliculator;
+public class Program
+{
+	public static void Main(string[] args)
+
+
+	{
+		int a;
+		int b;
```

Comment:
We usually don't define these here, but on the place where we start using the variables.